samples/snippets/publisher.py

```diff
@@ -189,7 +189,7 @@ def publish_messages_with_batch_settings(project_id: str, topic_id: str) -> None
     # or 1 KiB of data, or 1 second has passed.
     batch_settings = pubsub_v1.types.BatchSettings(
         max_messages=10,  # default 100
-        max_bytes=1024,  # default 1 MiB
+        max_bytes=1024,  # default 1 MB
```

Comment:
1 MiB should be the correct value here? The docstring above also says `The default value is 1 MiB`.

Reply:
1MB is correct, the docstring above was wrong

Reply:
Thanks @acocuzzo for double-checking. 

Java seems to default to 1 KB: 

https://github.com/googleapis/java-pubsub/blob/3709b2eb842ac014203680c1af5b081c7a952e2f/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java#L658

@hongalex Is it ok different libraries have different defaults?

Reply:
@hannahrogers-google  has submitted https://github.com/googleapis/java-pubsub/pull/908